developer_workflow.md

```diff
@@ -0,0 +1,40 @@
+# Developer Workflow
+
+We rely on feedback from each other to improve the quality of our code. Feedback is important at all levels of what we do. Our developer workflow allows us to work on our own, yet leverage meaningful feedback without taking too much time away from the rest of the team. Our workflow allows the whole team to be aware of what's happening and changing on the project. In our pull request process and (code reviews)[https://github.com/HaughtCodeworks/TeamGuide], we check on code quality and benefit from the team's expertise and project know-how without constantly pairing.
+
+## Development Steps
+
+* Claim a card from the project board. For more details on this step, visit our (project board)[https://github.com/HaughtCodeworks/TeamGuide] document.
+* Identify what needs to be done to complete this card. What does it mean for a feature to be complete? (create and reference separate post on this)
+* Create a local feature branch based off master. Give your branch a short descriptive name based on the card on which you are working.
+* Determine an appropriate level of testing for your branch, and decide how you will test your work. (create and reference separate post on testing)
+* When you are finished with your feature, make sure the entire test suite passes and your work hasn't broken the build.
+* Verify everything works locally with real data. Play with the feature and ensure the user experience is good.
+* Add to the README as you develop, making note of any steps required for deployment and testing: migrations, running scripts, updates to the server, etc. See our (readme document)[https://github.com/HaughtCodeworks/TeamGuide] for more on this.
+* Consider if you need to break your changes into multiple commits. Once you've grouped your changes into logical commits, add a good commit message. Make sure that you are only committing the changes that make sense for this feature.
+* Push your branch to origin, and avoid including sensitive files in source control.
+* Submit a pull request on Github/BitBucket, and tag relevant reviewers from your team.
+* Push your branch to the integration environment after verifying no one else on the team is using it. Make sure everything works in a true server environment.
```

Comment:
Let's clarify the end here to something like:  Test the functionality on the integration to make sure your feature works.

We don't want devs to have to test everything on integration unless their task touched everything.